_assets/docs/DEVELOPMENT_GUIDELINES.md

```diff
@@ -0,0 +1,11 @@
+This document aims to group all general guidelines to have in mind when you work with status-go.
+
+
+### /geth/ path
+
+Some status-go code lives inside `/geth` path, however this path no longer makes sense and we should get rid of it. Please don't store your new packages on `/geth` use the root path of the repo instead.
+
+
+### Server as standalone app
+
+`status-go` is used both for mobile devices and also on our nodes. We should try to split this in order to not overload single repository.
```

Comment:
There is almost no overload from my POV. We have most of the codebase shared, and only two  small "entrance" packages (`cmd/statusd` and `lib`) for server and mobile respectively.
I'm not sure what problem splitting this into two repositories will solve.

Reply:
discussion on this topic comes from [this issue](https://github.com/status-im/status-go/issues/846)

Reply:
- signals are completely ignored when statusd is used
- rpc interceptor doesn't make any sense for statusd
- shhext is not necessary with statusd
- different sources of defaults for config files and statusd

i am pretty sure there are more things

Reply:
also this must be tracked as a simple issue on github, rather then guideline